modules/veteran/app/controllers/veteran/application_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module VeteranApi
+  class ApplicationController < ::OpenidApplicationController
+    skip_before_action :set_tags_and_extra_content, raise: false
+    skip_before_action :authenticate
```

Comment:
If you're just going to to skip authenticate I don't think you need to inherit from `OpenIdApplicationController`

Reply:
This has been changed